packages/devtools_app/lib/src/shared/console/widgets/evaluate.dart

```diff
@@ -315,7 +308,19 @@ class ExpressionEvalFieldState extends State<ExpressionEvalField>
     try {
       // Response is either a ErrorRef, InstanceRef, or Sentinel.
       final isolateRef = serviceManager.isolateManager.selectedIsolate.value;
-      final response = await evalService.evalAtCurrentFrame(expressionText);
+      final Response response;
+      if (serviceManager.isMainIsolatePaused) {
```

Comment:
Is it possible for the main isolate to be paused but the selectedIsolate to be unpaused? Shouldn't we be evaluating in the selected isolate, not the main (in the event that these are different)?